lib/core/viewmodels/dashboard_viewmodel.dart

```diff
@@ -249,31 +249,40 @@ class DashboardViewModel extends FutureViewModel<Map<PreferencesFlag, int>> {
   bool isSameDay(DateTime a, DateTime b) =>
       a.year == b.year && a.month == b.month && a.day == b.day;
 
-  Future<void> startDiscovery(BuildContext context) async {
-    if (await _settingsManager.getString(PreferencesFlag.discoveryDashboard) ==
+  /// Start discovery is needed
+  static Future<void> startDiscovery(BuildContext context) async {
+    final SettingsManager _settingsManager = locator<SettingsManager>();
```

Comment:
Why is this now static? Was it because of the dual dashboards loaded at the same time? Should we try to just have one dashboard viewmodel instance loaded at all time instead or is there a deeper-rooted problem that prevent to do this solution.

Reply:
It's reducing the instantiated elements (e.g. no need to create the `_todayDateEvents` array if aren't going to use it) so speed up the loading time but also resolve the duplicated DashboardViewModel instance and so resolve the discovery bug 😄 

This is the best solution I have found but if you have an idea go ahead 😄

Reply:
Okay great! if it also speed up loading I'm all for that 😄